SU-SWS/stanford_help
# 7.x 1.x make files private

# READY FOR REVIEW

# Summary
Because Help pages were to be available only to site owners and maybe editors, the images and files attached should also be available only to the site owners, but alas, that wasn't the case. These images and files were publicly accessible if you had the URL. This PR remedies the situation. Drawing from similar work already completed on [Stanford Private Page](https://github.com/SU-SWS/stanford_private_page), this moves the files and images from the public directories into private directories and updates the URLs for files inserted into body text.

# Needed By (Date)
Ask John

# Urgency
John is pinging me!

# Steps to Test
Pull this branch (7.x-1.x-make-files-private) of Stanford Help onto a site
On a Stanford Help page insert an image and text file.
Run the database updates (drush updb)
Verify both image and text file still exist and are located in the site's private directory

# Affected Projects or Products
- Yes. JSE in particular, and other sites as well. See [John's report](https://stanfordits.atlassian.net/browse/SOE-2632?focusedCommentId=72397&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-72397).

# Associated Issues and/or People
- JIRA ticket
https://stanfordits.atlassian.net/browse/SOE-2632

- Any other contextual information that might be helpful 
See work on [Stanford Private Page](https://github.com/SU-SWS/stanford_private_page/blob/7.x-2.x/stanford_private_page.install#L63) 

- Anyone who should be notified?
@andydunmire 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
